src/helpers/ariaAppHider.js

```diff
@@ -1,37 +1,35 @@
-let globalElement = typeof document !== 'undefined' ? document.body : null;
-
 function validateElement (appElement) {
-  if (!appElement && !globalElement) {
-    throw new Error('react-modal: You must set an element with `Modal.setAppElement(el)` to make this accessible');
+  if (!appElement) {
+    throw new Error('react-modal: Setting an getAppElement function is required');
   }
 }
 
-export function setElement (element) {
-  let newElement = element;
-  if (typeof newElement === 'string') {
-    const el = document.querySelectorAll(element);
-    newElement = 'length' in el ? el[0] : el;
-  }
-  globalElement = newElement || globalElement;
-  return globalElement;
-}
-
 export function hide (appElement) {
   validateElement(appElement);
-  (appElement || globalElement).setAttribute('aria-hidden', 'true');
+  if (Array.isArray(appElement)) {
+    appElement.forEach((ae) => {
+      ae.setAttribute('aria-hidden', 'true');
+    });
+  } else {
+    appElement.setAttribute('aria-hidden', 'true');
+  }
 }
 
 export function show (appElement) {
   validateElement(appElement);
-  (appElement || globalElement).removeAttribute('aria-hidden');
+  if (Array.isArray(appElement)) {
+    appElement.forEach((ae) => {
+      ae.removeAttribute('aria-hidden');
+    });
+  } else {
+    appElement.removeAttribute('aria-hidden');
+  }
 }
```

Comment:
`show()` has the same effect if it sets `ae.setAttribute('aria-hidden', 'false');`, so `hide()` and `show()` could be`toggle(appElement : DOM, value : bool)`.